twitchio/abcs.py

```diff
@@ -162,7 +162,7 @@ async def clear(self):
     async def slow(self, time: int=30):
         """|coro|
 
-        Method which sends a .slow to Twitch and sets the channel to a slowmode slowmode of a specific time.
+        Method which sends a .slow to Twitch and sets the channel to a slowmode of a specific time.
```

Comment:
I'd change it to  
A method that sends a ".slow" to Twitch putting the channel into slowmode for a given time.

Reply:
Or something to that effect

Reply:
Updated, I'm not very good with descriptions lmao